src/packages/uploader/uploader.tsx

```
@@ -383,6 +387,85 @@ const InternalUploader: ForwardRefRenderFunction<
   }
 
+  const handlePaste = useCallback(
+    (event: ClipboardEvent) => {
+      if (!enablePasteUpload || disabled) return
+
+      const clipboardData = event.clipboardData
+      if (!clipboardData) return
+
+      const files: File[] = []
+
+      if (clipboardData?.items && clipboardData.items.length) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)